force-app/main/default/classes/DuplicateSeasonHandler.cls

```diff
@@ -0,0 +1,47 @@
+public with sharing class DuplicateSeasonHandler {
+
+
+    public static void checkDuplicates(List<Season__c> seasons) {
+ 		
+        System.debug('###Season__c from trigger: ' + seasons);      
+        
+        List<Id> tvSerieIds = new List<Id>();
+        for(Season__c s : seasons) {
+            tvSerieIds.add(s.TV_Serie__c);
+        }
+        System.debug('###TV_Serie_Id from trigger: ' + tvSerieIds);
+        
+		Map<String,Id> mapSeasons = new Map<String,Id>();
+        // takes seasons that tv series id are in input values
+        List<Season__c> seasonsInDB = [SELECT Id,Sesaon_Name__c,TV_Serie__c FROM Season__c where TV_Serie__c IN :tvSerieIds];
```

Comment:
before that select check if tvSerieIds is not empty

Reply:
Done